cmd/exec.go

```diff
@@ -75,19 +93,7 @@ func execRun(cmd *cobra.Command, args []string) error {
 		return errors.Wrap(err, "Unable to extract right token output from AWS Assume Web identity")
 	}
 
-	// our calculated awsEnvVars take precedence
-	envVars := append(
-		getAWSEnvVars(assumeRoleOutput),
-		os.Environ()...,
-	)
+	envVars := getAWSEnvVars(assumeRoleOutput, awsOIDCConfig)
```

Comment:
```suggestion
	envVars := append(
	      getAWSEnvVars(assumeRoleOutput, awsOIDCConfig),
	      os.Environ()...,
	 )
```
I think only exec should get all the env variables while env only gets the aws specific ones.